custom/icds_reports/views.py

```diff
@@ -1773,7 +1773,7 @@ def valid_state_names(self):
 
 
 @location_safe
-@method_decorator([api_auth, csrf_exempt, toggles.ICDS_NIC_INDICATOR_API.required_decorator()], name='dispatch')
+@method_decorator([api_auth, toggles.ICDS_NIC_INDICATOR_API.required_decorator()], name='dispatch')
```

Comment:
is the toggle necessary?

Reply:
There is no other permission for these APIs that we can use to assign/deassign for mobile workers. Without this, all mobile workers could access the API.

Reply:
Alternative is to create `API Access Permission` with list of APIs, that is similar to `View Report` permission.